src/Microsoft.TestPlatform.CoreUtilities/PublicAPI/net451/PublicAPI.Shipped.txt

```diff
@@ -1,15 +0,0 @@
-﻿#nullable enable
```

Comment:
did this have its own file for net462? All the other files seem renamed, not removed.

Reply:
Or is that because net45 was used?